ntoskrnl/config/i386/cmhardwr.c

```diff
@@ -357,6 +366,7 @@ CmpInitializeMachineDependentConfiguration(IN PLOADER_PARAMETER_BLOCK LoaderBloc
             ConfigData.ComponentEntry.AffinityMask = AFFINITY_MASK(i);
             ConfigData.ComponentEntry.Identifier = Buffer;
 
+#if defined(_M_IX86)
             /* Check if the CPU doesn't support CPUID */
             if (!Prcb->CpuID)
```

Comment:
We don't support CPUs without CPUID, so this should be removed completely

Reply:
I didn't touched that code anyway :wink: